app/controllers/HomeController.scala

```diff
@@ -90,9 +115,24 @@ class HomeController @Inject() (
                                                                                                  taxCalculationStateCyMinusTwo
                                                                                                )
             shutteringMessaging                                                             <- featureFlagService.get(NpsShutteringToggle)
-
+            bannerAlert                                                                     <- featureFlagService.get(AlertBannerToggle)
```

Comment:
Do we need a toggle on the banner itself?
I would a toggle in each of the feature available in the banner

Reply:
Done